src/components/analyses/utils.js

```diff
@@ -210,6 +210,22 @@ const canShare = (selectedAnalyses) => {
     );
 };
 
+const ANALYSIS_TERMINAL_STATES = [
+    analysisStatus.CANCELED,
+    analysisStatus.COMPLETED,
+    analysisStatus.FAILED,
+    analysisStatus.REMOVED,
```

Comment:
I'm not 100% sure if these are all the terminal states from [here](https://github.com/cyverse-de/sonora/blob/master/src/components/models/analysisStatus.js)

Reply:
LGTM 👍🏽